src/dump.c

```diff
@@ -1339,8 +1348,7 @@ extern void jl_get_system_hooks(void);
 extern void jl_get_uv_hooks();
 
 // Takes in a path of the form "usr/lib/julia/sys.ji", such as passed in to jl_restore_system_image()
-DLLEXPORT
-const char * jl_get_system_image_cpu_target(const char *fname)
+DLLEXPORT const char *jl_get_system_image_cpu_target(const char *fname)
```

Comment:
i hadn't noticed this function before. that explains why i keep getting warnings from gdb about attempting to load sys.dll whenever i'm using julia-debug.exe